algorithms/kernel/pca/transform/pca_transform_helper.h

```diff
@@ -0,0 +1,67 @@
+/* file: pca_transform_helper.h */
+/*******************************************************************************
+* Copyright 2014-2020 Intel Corporation
```

Comment:
minnor: new file must have copyright this year:

```c++
* Copyright 2020 Intel Corporation
```

Reply:
Thanks!